### PR TITLE
consolidating files

### PR DESCRIPTION
**Pull request by jenniferward**
Marking joint files with <jointfile> for files that should be the same no matter what language.
Consolidating and rearranging other files.

### CODE REVIEW

File: default/en/cat_printed.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
-# When to input a new record (printed music)
+## Scope of printed music in RISM
+
+RISM seeks to document printed music from the beginnings of music printing through around 1900.
+
+In the course of the 20th century and up to today, we see a sharp increase in commercial music publishing and
+distribution. The amount of music considered "rare" or undocumented elsewhereis much less when compared to previous
```
Comment (ahankinson) [inline default/en/cat_printed.md:6]:
```suggestion
distribution. The amount of music considered "rare" or undocumented elsewhere is much less when compared to previous
```

File: default/en/cat_printed.md [hunk 2]
```diff
@@ -66,3 +79,68 @@ PLATTNER à ROTTERDAM." **New record required** (different publication status).
    field "Material held" please note which symphony (livre) you have.
    2. Option 2, a better solution, but takes more time: **Notify the Editorial Center** about this record and we will split
    the record into 3, one record for each symphony. Then, add your holdings to the record for the symphony you have.
+
+## Standardized titles for printed music
+
+   **Distinctive titles**
```
Comment (ahankinson) [inline default/en/cat_printed.md:85]: These should probably be level-3 headings:

```
### Distinctive titles
```

And not indented.

File: default/en/cat_printed.md [hunk 3]
```diff
@@ -66,3 +79,68 @@ PLATTNER à ROTTERDAM." **New record required** (different publication status).
...
+      - Standardized title: Vezzo di perle musicali
+
+   - RISM A/I: B 3781
+      - Title on source: Novo giardino de concerti a quattro voci, per cantare a due chori con due voci, e due tromboni, o altri
+   stromenti, o voci, secondo la comodità de cantori ... nel quale li contengono alquante antifone del cantico della Beata
+   Virgine, di alcune solennità principali del anno, & altri motetti, con il basso principale per l'organo, opera
+   undecima  
+      - Standardized title: Novo giardino de concerti
+
+   **Generic titles**
```
Comment (ahankinson) [inline default/en/cat_printed.md:120]: Level 3 heading

File: default/it/cat_printed.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+**Statuto editoriale diverso**: Un frontespizio originale è stato cancellato e rimpiazzato. Una nuova etichetta copre i dati di pubblicazione, distribuzione, produzione ecc. Le pagine originali sono state pubblicate in una nuova coperta che reca informazioni più recenti di quelle riportate sul frontespizio. È stato aggiunto un frontespizio di una serie.
+
+-
+
+**Numero di lastra e/o numero editoriale diverso**.
+
+**Non creare una nuova scheda**  se il tuo oggetto mostra una delle differenze seguenti. (Se il tuo oggetto mostra più di una di queste differenze, devi decidere se è necessaria una nuova scheda.)
+
+-
```
Comment (ahankinson) [inline default/it/cat_printed.md:36]: I think the formatting of this list is messed up

File: default/it/help_opera_houses.md [hunk 5]
```diff
@@ -96,15 +96,15 @@
...
  Várszínház (Buda) |
 | Caserta:  
-  
-  
-  
-  
-  
-  
-  
+
```
Comment (ahankinson) [inline default/it/help_opera_houses.md:99]: These blank spaces should be removed

File: default/pl/cat_printed.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+
+**Inny status wydania**: Usunięto lub wymieniono oryginalną stronę tytułową. Na oryginalnej strefie wydania, dystrybucji, produkcji itd. znajduje się nowa etykieta. Oryginalne wydanie nut wydano w okładce nowego wydawcy zawierającej więcej informacji niż podano na stronie tytułowej. Strona tytułowa serii jest nowa.
+
+-
+
+**Inna etykieta i/lub numery wydawcy.**
+
+**Nie należy tworzyć nowego rekordu,** jeżeli opisywana pozycja wykazuje jedną z poniższych różnic (jeżeli opisywana pozycja posiada _więcej niż jedną_ z poniższych różnic, należy zdecydować, czy nowy rekord jest konieczny.)
+
+-
```
Comment (ahankinson) [inline default/pl/cat_printed.md:36]: This list is also formatted incorrectly

File: default/en/cat_printed.md (hunk 2 above)
Comment (jenniferward) [reply default/en/cat_printed.md:85]: Yes, thanks - in this PR, I only joined files. I haven't edited them yet.

File: default/it/cat_printed.md (hunk 4 above)
Comment (jenniferward) [reply default/it/cat_printed.md:36]: Probably. Will do that in a separate phase.

File: default/it/help_opera_houses.md (hunk 5 above)
Comment (jenniferward) [reply default/it/help_opera_houses.md:99]: Will do that in a separate phase. I just changed the name here. Hopefully this file will disappear completely in the future (it is a "joint file" to be shared)

File: default/pl/cat_printed.md (hunk 6 above)
Comment (jenniferward) [reply default/pl/cat_printed.md:36]: I just changed the file name here. I will edit in a separate phase.

**Pull request merged**